Source/LibationWinForms/Form1.Liberate.cs

```diff
@@ -11,12 +11,18 @@ public partial class Form1
 
 		//GetLibrary_Flat_NoTracking() may take a long time on a hugh library. so run in new thread 
 		private async void beginBookBackupsToolStripMenuItem_Click(object sender, EventArgs e)
-			=> await Task.Run(() => processBookQueue1.AddDownloadDecrypt(ApplicationServices.DbContexts.GetLibrary_Flat_NoTracking()
+		{
+			SetQueueCollapseState(false);
```

Comment:
Great minds think alike. I was going to suggest this too but I forgot to mention it when I was writing that email